src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
```

Comment:
Creating a RightTriangle, you should avoid passing expressions directly into the constructor. Instead, create separate variables for 'firstLeg' and 'secondLeg' before passing them to the constructor for better code readability.